frontend/app/views/fragments/offer/option.scala.html

```diff
@@ -0,0 +1,10 @@
+@(title: String, offer: String, fullPrice: String, rightGutter: Boolean = false, note: String = "")
```

Comment:
`note` should be an Option of a string, not an empty string. To keep @wpf500 happy